.circleci/src/commands/install_node_modules.yml

```diff
@@ -2,12 +2,13 @@ description: Restores and saves the node_modules directories of the npm workspac
 steps:
   - restore_cache:
       keys:
-      - v1-node-modules-{{checksum "~/project/package-lock.json" }}
+      - v1-node-modules-{{ checksum "~/project/package-lock.json" }}
+      - v1-node-modules
   - run:
       name: Install node dependencies for npm workspaces
-      command: "[ ! -d node_modules ] && npm ci --ignore-scripts --loglevel warn --yes || echo package.json and package-lock.json unchanged. Using cache."
+      command: npm ci --ignore-scripts
```

Comment:
I still don't get why we use `--ignore-scripts`. Do you know why? 🤔

Reply:
Otherwise it tries to run the following scripts automatically as they are configured as postinstall which fails if the current working directory is NOT `administration`:
``` json
    "generate-graphql": "graphql-codegen --config graphql-codegen.yml",
    "generate-protobuf": "protoc -I ../specs --es_out=src/generated --es_opt=target=ts --plugin=protoc-gen-es=../node_modules/.bin/protoc-gen-es ../specs/card.proto",
    "postinstall": "npm run generate-protobuf && npm run generate-graphql",
```
See https://github.com/digitalfabrik/entitlementcard/blob/main/administration/package.json#L121
I tried removing it [here](https://app.circleci.com/pipelines/github/digitalfabrik/entitlementcard/7759/workflows/72a6204a-4ea1-4228-8d59-644a29c14d08/jobs/52952).

Reply:
I don't think the problem there is that `administration` is not the working directory, but rather that the directory "administration/src/generated" does not exist when running "generate-protobuf", and protoc does not seem to automatically create the directory.
This could be fixed by adjusting the run config of  "generate-protobuf" to 
```json
    "generate-protobuf": "mkdir -p src/generated && protoc -I ../specs --es_out=src/generated --es_opt=target=ts --plugin=protoc-gen-es=../node_modules/.bin/protoc-gen-es ../specs/card.proto",
```

What I still don't get is: Why does the `prepare` script in "frontend/build-configs" get executed if the `--ignore-scripts` flag is used 🤔